source/hint_list.cpp

```diff
@@ -2067,6 +2081,51 @@ void HintTable_Init() {
                        Text{"the #treasure chest game# grants", /*french*/"la #chasse aux trésors# donne", /*spanish*/"#el Cofre del Tesoro# premia"}
   );
 
+    hintTable[MARKET_TREASURE_CHEST_GAME_ITEM_1] = HintText::Sometimes({
+                       //obscure text
+                       Text{"#gambling once# grants",               /*french*/"#parier# donne [1]",                     /*spanish*/"#los juegos de azar# revelan [1]"},
+                       Text{"the #first or second chest# contains",               /*french*/"#parier# donne [1]",                     /*spanish*/"#los juegos de azar# revelan [1]"},
```

Comment:
#parier une fois# donne
le #premier ou deuxième coffre à jeu# contient

la #première salle# de la chasse aux trésors contient

change "chest" to "game chest" for the obscure hints